doc/nrf/ug_nrf52.rst

```diff
@@ -131,6 +138,41 @@ The nRF52 Series devices support running another protocol in parallel with the n
 The :ref:`Multi-Protocol Service Layer (MPSL) <nrfxlib:mpsl>` library provides services for multi-protocol applications.
 
 
+.. |note| replace:: There is currently no support for upgrading the bootloader (:doc:`mcuboot:index`) on nRF52 Series devices.
+
+.. fota_upgrades_start
+
+FOTA upgrades
+*************
+
+|fota_upgrades_def|
+FOTA upgrades can be used to replace the application.
+
+.. note::
+   |note|
+
+To perform a FOTA upgrade, complete the following steps:
+
+1. Make sure that your application supports FOTA upgrades.
+      To download and apply FOTA upgrades, the following requirements apply:
+
+      * You must enable :option:`CONFIG_MCUMGR_SMP_BT` in your application.
+      * |fota_upgrades_req_mcuboot|
```

Comment:
Missing requirement off integration of mcumgr modules in the appliation code:
enable CONFIG_MCUMGR_CMD_OS_MGMT and register OS management command set in the application C code (`os_mgmt_register_group();`)

enable CONFIG_MCUMGR_CMD_IMG_MGMT and register image managemt command set inthe application C code
(`img_mgmt_register_group();`)

Application must initialize initialize the Bluetooth mcumgr transport (`using smp_bt_register()`)
Application shall advertise SMP service UUID 8D53DC1D-1DB7-4CD3-868B-8A527460AA84 in BT advertisement data
Refer to zephyr/samples/subsys/mcumgr/smp_svr for code sample on above requirements.

Reply:
Thanks! Can you please check my update @nvlsianpu & @hakonfam ?

![image](https://user-images.githubusercontent.com/11227796/77773067-648b5d80-7049-11ea-9e9f-50c07560abb5.png)

Reply:
This is correct to my knowledge, but I would appreciate @nvlsianpu 's final approval.

Reply:
> enable CONFIG_MCUMGR_CMD_IMG_MGMT and register image managemt command set inthe application C code
(img_mgmt_register_group();)

^^this is not included. I would extend points `a` and` b` by this.

Reply:
Oops! Added now.